model_signing/serialization/conftest.py

```diff
@@ -102,3 +104,12 @@ def deep_model_folder(tmp_path_factory):
         file.write_text(f"This is file f{i}.")
 
     return model_root
+
+@pytest.fixture
+def symlink_model_file(tmp_path_factory: pytest.TempPathFactory) -> pathlib.Path:
+    """A model with a symlink to a single file."""
+    source_file = tmp_path_factory.mktemp("source") / "file"
+    source_file.write_bytes(test_support.KNOWN_MODEL_TEXT)
```

Comment:
I could also use the sample_model_file fixture directly to create the source file